server/src/routes/swagger.ts

```diff
@@ -0,0 +1,32 @@
+// SPDX-FileCopyrightText: 2022 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+/**
+ * This file is used as a base for the swagger complition in the index.js file of routes.
+ */
+
+export const definition: object = {
+  openapi: '3.0.1',
+  info: {
+    version: '1.0.0',
+    title: 'APIs Document',
+    description: 'Documentation of Aalto Grades Backend API',
+    termsOfService: '',
+    license: {
+      name: 'MIT',
```

Comment:
I would write `Expat (MIT)` since this doesn't appear to be an SPDX identifier. https://www.gnu.org/licenses/license-list.html#Expat
> Some people call this license “the MIT License,” but that term is misleading, since MIT has used many licenses for software. It is also ambiguous, since the same people also call the [X11 license](https://www.gnu.org/licenses/license-list.html#X11License) “the MIT License,” failing to distinguish them. We recommend not using the term “MIT License.”